### PR TITLE
gamejolt is shit lol7

### PR DESCRIPTION
**Pull request by NimuFurrySoftUwU**

### CODE REVIEW

**Comment by SrtHero278** [comment]
wrong repository

**Comment by Joalor64GH** [comment]
tf is wrong with this kid

**Review by ManMaster707 — CHANGES_REQUESTED** [review]
This kid is CERTAINLY a troll, and if YOU, @juandavid45, are reading this, please, just stop, you're not making anyone's life funnier or easier, you've done this everywhere, even in Kade Engine's pull requests, so stop trying to insert your friends and you in the engine, think about your life decisions, and don't touch github ever again.

**Comment by SonicProYT** [comment]
oh god the same guy who destroyed the credits

HE DID IT AGAIN

**Review by charlesisfeline — COMMENTED** [review]
SEE?!?
HE RUINED THE CREDITS ONCE AGAIN

**Comment by voxovistired** [comment]
Never touch GitHub again. Please, it's the best for you and this world.

**Review by nahiassis — COMMENTED** [review]
hell nah
goofy ahh thing

**Comment by CrusherMods** [comment]
Yo @juandavid45 
please do this its for your own good
image

**Comment by Wither362** [comment]
Wtf is wrong with you, also you need the icons for the credits, so this code is literally wrong

**Comment by Joalor64GH** [comment]
that's why i blocked the dummy

**Comment by Swapper1** [comment]
I should mention this here. On the Kade Engine's [pull requests page](https://github.com/KadeDev/Kade-Engine/pull/2925), the code was taken from Bob and Bosip mod, where the Story Menu is replace by the Main Menu.
https://github.com/ash237/vs-Bob-and-Bosip/blob/main/source/MainMenuState.hx

**Comment by SonicProYT** [comment]
should i ask ShadowMario to close this pr and block this kid from the psych repo

this kid ruins psych

**Comment by SonicProYT** [comment]
(by kid, i mean, the child who made this pr)

**Comment by Raltyro** [comment]
why the fuck are you making another stupid rep

**Comment by Theoyeah** [comment]
wrong repo you motherfucker

File: source/CreditsState.hx [hunk 1]
```diff
@@ -81,7 +81,12 @@ class CreditsState extends MusicBeatState
 		#end
 
 		var pisspoop:Array<Array<String>> = [ //Name - Icon name - Description - Link - BG Color
-			['Psych Engine Team'],
+			['Vs Jefferson13n Team'],
```
Comment (Theoyeah) [inline source/CreditsState.hx:84]: Fatherless

**Review by Theoyeah — COMMENTED** [review]
I have your ip stupid kid

**Comment by CrusherMods** [comment]
YO IT GOT CLOSED AGAIN
LETS PARTY YESSSSSSS

**Comment by Joalor64GH** [comment]
> YO IT GOT CLOSED AGAIN LETS PARTY YESSSSSSS

F***IN HELL YEAHHHH

File: source/CreditsState.hx (hunk 1 above)
Comment (Gibberditoslumber) [reply source/CreditsState.hx:84]: DANG NO MERCIY

**Comment by REALHaydenGaming** [comment]
YEAH, FATHERLESS RETARD GETS HIS PULL REQUEST DOWN

**Pull request closed without merging**